src/infrastructure/figma/transformers/utils.ts

```diff
@@ -23,30 +23,33 @@ export const buildDesignUrl = ({
 };
 
 /**
- * Builds a Live Embed URL to a Figma design given Figma file/node metadata.
- * Inspect URL is used as this is the preferred mode of display in Jira.
- * @see https://www.figma.com/developers/embed
+ * Builds an Inspect Mode URL to a Figma design given Figma file/node metadata.
  */
-export const buildLiveEmbedUrl = ({
```

Comment:
**note:** The PR deletes a Figma File name from URLs and changes the order of function declarations to put dependent functions after their dependencies.